app/emails/templates/team_lead_added.md

```diff
@@ -0,0 +1,19 @@
+Hello,  
+  
+You've been added as a team lead of {{ team_name }}. This means you can:  
+  
+* add and remove team members
+* add and remove other team leads
+* set permissions for each member
+  
+[Sign in]({{ frontend_url }}/login) to the Digital Marketplace to view your team's opportunities on your dashboard.  
+  
+To manage your team, go to the menu in the top right of the screen and select 'Teams and people'.  
```

Comment:
Should we link directly to `Teams and People`?

Reply:
changed to link